imports/react-ui/settings/knowledgebase/containers/utils.js

```diff
@@ -0,0 +1,22 @@
+import { Meteor } from 'meteor/meteor';
+import { FlowRouter } from 'meteor/kadira:flow-router';
+import Alert from 'meteor/erxes-notifier';
+
+export const saveCallback = (paramsDic, addMethodName, editMethodName, item, redirectPath) => {
+  let methodName = `knowledgebase.${addMethodName}`;
+  let params = { ...paramsDic };
+
+  if (item && item._id) {
+    methodName = `knowledgebase.${editMethodName}`;
+    params = { _id: item._id, ...paramsDic };
+  }
+
+  Meteor.call(methodName, params, error => {
```

Comment:
Since we use `validated-method`, we need to call a method in their ways. https://github.com/meteor/validated-method

Reply:
done